engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -997,3 +997,15 @@ BEGIN
     DECLARE CONTINUE HANDLER FOR 1061 BEGIN END; SET @ddl = CONCAT('ALTER TABLE ', in_table_name); SET @ddl = CONCAT(@ddl, ' ', ' ADD KEY ') ; SET @ddl = CONCAT(@ddl, ' ', in_index_name); SET @ddl = CONCAT(@ddl, ' ', in_key_definition); PREPARE stmt FROM @ddl; EXECUTE stmt; DEALLOCATE PREPARE stmt; END;
 
 CALL `cloud`.`IDEMPOTENT_ADD_KEY`('i_user_ip_address_state','user_ip_address', '(state)');
+
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'AlmaLinux 9', 'KVM', 'default', 'AlmaLinux 9');
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'CentOS 9', 'KVM', 'default', 'CentOS 9');
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'Oracle Linux 9', 'KVM', 'default', 'Oracle Linux 9');
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'Red Hat Enterprise Linux 9', 'KVM', 'default', 'Red Hat Enterprise Linux 9');
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'Rocky Linux 9', 'KVM', 'default', 'Rocky Linux 9');
+
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'AlmaLinux 9', 'VMware', '7.0.3.0', 'otherLinux64Guest');
```

Comment:
@rohityadavcloud For vmware, should we add `otherLinux64Guest` for OS which doesn't return ID. I tried to log guest OS descriptors for VMware 7u3 and can't find some of the OSes.
[vmware-guestos-descriptors.txt](https://github.com/apache/cloudstack/files/10371810/vmware-guestos-descriptors.txt)

But could see them listed here, https://www.vmware.com/resources/compatibility/search.php?deviceCategory=software

For XCP-ng/Xen, I don't see any EL9 distro here, https://docs.citrix.com/en-us/citrix-hypervisor/system-requirements/guest-os-support.html

Reply:
LGTM @shwstppr can we move this to top of the sql (or move the sql related down) - where we add support for vSphere 8; so that these mappings are also copied for vSphere 8.